src/main/java/it/rate/webapp/services/InterestService.java

```diff
@@ -64,6 +64,9 @@ interest, getDistanceToNearestPlace(usersCoords, interest.getPlaces())))
 
   private double getDistanceToNearestPlace(CoordinatesDTO usersCoords, List<Place> places) {
     double minDistance = Double.MAX_VALUE;
+    if (places.isEmpty()) {
+      return minDistance;

```

Comment:
I guess that returning 179769313486231590772930519078902473361797697894230657273430081157732675805500963132708477322407536021120113879871393357658789768814416622492847430639474124377767893424865485276302219601246094119453082952085005768838150682342462881473913110540827237163350510684586298239947245938479716304835356329624224137216 is not exactly what this method should be doing.